html/host.html

```diff
@@ -3,66 +3,100 @@
     <meta charset="utf-8"/>
     <link href="/assets/stylesheets/main.css" rel="stylesheet">
     <link href="/assets/stylesheets/host.css" rel="stylesheet">
+    <script type="module">
```

Comment:
So *where* in the `js` directory should it be?

Well, the main thing you're doing here is reacting to the `DOMContentLoaded` event.  That event (and basically all browser events) is already handled in `js/host/main.js`.  Change the code:

```javascript
document.addEventListener("DOMContentLoaded", nlwManager.init);
```

to

```javascript
document.addEventListener("DOMContentLoaded", () => {
  my_cool_code;
  nlwManager.init()
});
```

Reply:
Done!